Dockerfile

```diff
@@ -2,21 +2,28 @@ FROM ubuntu:14.04
 
 WORKDIR /
 
-RUN apt-get update
-# Install pip's dependency: setuptools:
-RUN apt-get install -y python python-dev python-distribute python-pip
-RUN pip install ansible
-
 ADD bootstrap/playbooks/celery.yaml /celery.yaml
 ADD resources /resources
 ADD templates /templates
 ADD run.sh /run.sh
 
+RUN apt-get update
+# Install pip's dependency: setuptools:
+RUN apt-get install -y python python-dev python-distribute python-pip \
+    libyaml-dev vim libffi-dev libssl-dev
+RUN pip install ansible
 
 RUN apt-get install -y libffi-dev libssl-dev
+
 RUN pip install https://github.com/Mirantis/solar/archive/master.zip
 RUN pip install https://github.com/Mirantis/solar-agent/archive/master.zip
 
 RUN ansible-playbook -v -i "localhost," -c local /celery.yaml --tags install
 
+RUN pip install -U setuptools>=17.1
+RUN pip install -U python-fuelclient
```

Comment:
I dislike that we will have fuelclient and hiera inside solar container. I understand though why is that. Maybe we have some better ideas / solutions instead ? f2s requires some manual work, maybe we could instead create separate image with base on `solar` ?